.travis.yml

```diff
@@ -1,40 +0,0 @@
-dist: bionic
-
-addons:
-  chrome: stable
-
-language: ruby
-
-matrix:
-  include:
-    - rvm: 2.7.0
-      env: "RAILS_VERSION=6.0.2.2"
-    - rvm: 2.6.5
-      env: "RAILS_VERSION=6.0.2.2"
-    - rvm: 2.6.5
-      env: "RAILS_VERSION=5.2.4.2"
-    - rvm: 2.5.7
-      env: "RAILS_VERSION=5.2.4.2 BLACKLIGHT_API_TEST=true ENGINE_CART_RAILS_OPTIONS=\"--api --skip-git --skip-bundle --skip-listen --skip-spring --skip-yarn --skip-keeps --skip-action-cable --skip-coffee --skip-test\""
-    - rvm: jruby-9.2.8.0
-      env: "RAILS_VERSION=5.2.4.2 JRUBY_OPTS=\"-J-Xms512m -J-Xmx1024m\""
-  allow_failures:
-    - rvm: jruby-9.2.8.0
-  fast_finish: true
-
-before_install:
-  - google-chrome-stable --headless --disable-gpu --no-sandbox --remote-debugging-port=9222 http://localhost &
-
-env:
-  global:
-    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
-    - ENGINE_CART_RAILS_OPTIONS='--skip-git --skip-listen --skip-spring --skip-keeps --skip-action-cable --skip-coffee --skip-test'
-    - CC_TEST_REPORTER_ID=5042c7358c96b0b926088a4cda3e132fffe7a66ce8047cdb1dc6f0b4b6676b79
-
-before_script:
-  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
Is there a plan to integrate CodeClimate in the github actions build?  Looks like it isn't reporting in pre-github actions PRs so doesn't seem like a blocker for this PR.

Reply:
Yeah -- it seemed like it wasn't currently working, so 🤷‍♂️